sapl/parlamentares/views.py

```diff
@@ -403,6 +403,43 @@ class UpdateView(Crud.UpdateView):
         form_class = FrenteForm
 
 
+class FrenteParlamentarCrud(MasterDetailCrud):
+    model = FrenteParlamentar
+    parent_field = 'frente'
+    help_topic = 'frente_parlamentares'
+    public = [RP_LIST, RP_DETAIL]
+
+    class CreateView(MasterDetailCrud.CreateView):
+        form_class = FrenteParlamentarForm
+
+        def get_context_data(self, **kwargs):
+            context = super().get_context_data(**kwargs)
+            context['subnav_template_name'] = ''
+            return context
+
+        def get_initial(self):
+            self.initial['frente'] = Frente.objects.get(pk=self.kwargs['pk'])
+            return self.initial
+
+    class UpdateView(MasterDetailCrud.UpdateView):
+        form_class = FrenteParlamentarForm
+
+        def get_context_data(self, **kwargs):
+            context = super().get_context_data(**kwargs)
+            context['subnav_template_name'] = ''
+            return context
+
+    class DetailView(MasterDetailCrud.DetailView):
+        def get_context_data(self, **kwargs):
+            context = super().get_context_data(**kwargs)
+            context['subnav_template_name'] = ''
+            return context
+
+    class ListView(MasterDetailCrud.ListView):
+        layout_key = 'FrenteParlamentarList'
+        ordering = ('-cargo__cargo_unico', 'parlamentar')
+
+
```

Comment:
Ocultar a subnav na listagem.
```suggestion
        def get_context_data(self, **kwargs):
            context = super().get_context_data(**kwargs)
            context['subnav_template_name'] = ''
            return context


```